### DISCUSSION TITLE
How can I place the tab node tree to the left side please?

### DISCUSSION THREAD

**Discussion by bd-bord1** [Able-Bear-Software/tab-nodes-tree#91] [Q&A] [answered]
How can I place the tab node tree to the left side please?

**Comment by Tab-Nodes-Tree** [comment, marked as answer]
@bd-bord1 You can find the instructions in the [Q&A](https://www.tab-nodes-tree.app/qna?q=change-side-panel-position). 
Let me know if you have any other questions!
